documentation/db-structure.puml

```diff
@@ -0,0 +1,86 @@
+@startuml
+!theme plain
+top to bottom direction
+skinparam linetype ortho
+
+entity organization {
+   _id: objectid
+   contactData: object
+   mediaData: object
+   organizationData: object
+   request: string
+   users: list
+   boards: list
+   __v: int32
+
+}
+
+class contactData {
+    email: string
+    firstName: string
+    lastName: string
+    middleName: string
+    phone: string
+    position: string
+}
+
+class organizationData {
+   certificate: string
+   dateOfRegistration: isodate
+   edrpou: int32
+   organizationName: string
+}
+
+class mediaData {
+   site: string
+   social: array
+}
+
+entity superadmin {
+   _id: objectid
+   email: string
+   password: string
+   role: string
+   __v: int32
+}
+
+entity user {
+   _id: objectid
+   email: string
+   middleName: string
+   organizationId: objectid
+   password: string
+   role: string
+   firstName: string
+   phone: string
+   status: string
+   lastName: string
+   position: string
+   __v: int32
+}
+
+entity board {
+    _id: objectid
+    name: string
+    organizationId: objectid
+    tasks: list
+    __v: int32
+}
```

Comment:
where is the connection board with columns?
![image](https://github.com/user-attachments/assets/f59a350a-afe6-4ec9-9296-e239e1b8ca99)

Does the task need to be linked to board Columns?

Reply:
That's the plan, but you can always suggest some other variation